src/components/ContentCardEditable.vue

```diff
@@ -47,24 +50,16 @@
 import Vue from 'vue'
 import SubjectTag from '@/components/SubjectTag.vue'
 import ContentCardEditorButton from '@/components/ContentCardEditorButton.vue'
-
 export default Vue.extend({
   components: {
     SubjectTag,
     ContentCardEditorButton
   },
   props: {
-    subjects: {
-      type: Array,
-      default: () => []
-    },
-    title: {
-      type: String,
-      default: ''
-    },
-    description: {
-      type: String,
-      default: ''
+    lesson: {
+      type: Object,
+      required: true,
+      default: () => {}
```

Comment:
```suggestion
    lesson: {
      type: Object as () => Lesson,
      required: true,
      default: () => {}
```
みたいなのが書けるはず（defaultがめんどそう）

Reply:
※ https://github.com/codeforjapan/StudyAtHome/pull/139#discussion_r439722205 に関連する